qubes-bug-report

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+import subprocess as sub
+import shlex
+import argparse
+import time
+import sys
+
+from os.path import expanduser
+
+#the term qube refers to a qubes vm
+
+def is_program_installed_in_qube( qube_name, command ):
+	is_installed = True
+
+	try:
+		shell_cmd = "qvm-run " + shlex.quote( qube_name ) + " --pass-io --no-color-output 'command -v " + command + "' &> /dev/null"
+		sub.check_call( shell_cmd, shell = True )
```

Comment:
Better not use `shell=True` and properly split command args array (place `'command -v ...'` as a single argument).